sonar-text-dotnet/src/IntegrationTests/SonarLint.Secrets.DotNet.NuGetPkg.Tests/SmokeTests.cs

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2021 SonarSource SA
```

Comment:
FYI I see the other copyrights have been updated to say `2021-2023`.
This one might have been missed because it is in a different format.